Toggle between Class.php and ClassTest.php with one shortcut

Starli0n opened the pull request:
Hi,

I added two features:
- Use one shortcut to switch from Class.php to ClassTest.php and vice versa.
- Do a matching between the class and the file.

For example my class `micClass` is matching the file `miClass.php`. 
Therefore, I had two (facultative) entries in `PHPUnit.sublime-settings`

```
{
    "prefix_file": "mi",
    "prefix_class": "mic"
}
```

Hope, this could help.

Starli0n committed:
Toggle between Class.php and ClassTest.php with one shortcut

stuartherbert commented:
Hi,

It isn't clear to me why this is needed? It sounds like you're creating code that isn't PSR0 or PSR4 compliant. If that's the case, I'm afraid that I can't accept your pull request.

Best regards,
Stu
--

Starli0n committed:
Fix issue #60

Starli0n commented:
Hi,

After almost 2 years, I came back on this PR as I was doing some cleaning on my repos.

I removed the commit that was not PSR0/PSR4 compliant. I did not know what it meant back in the days.

In this PR, I fixed the issue #60 and I offer the ability to toggle between Class.php and ClassTest.php with one shortcut.

Could you please review it?

Thanks
